### PR TITLE
add support for FX parameters

### PR DESCRIPTION
**Pull request by vladx1900**

### CODE REVIEW

File: src/PayU/Alu/Request.php [hunk 1]
```diff
@@ -203,6 +210,11 @@ private function transformObject2Array()
             );
         }
 
+        if (isset($this->fx)) {
```
Comment (drealecs) [inline src/PayU/Alu/Request.php]: `fx !== null` or `fx instanceof FX` would be more clear.

File: tests/PayU/Alu/RequestTest.php [hunk 2]
```diff
@@ -76,7 +76,9 @@ public function setUp()
 
         $card = new Card('5431210111111111', '11', 2016, 123, 'test');
 
-        $this->request = new Request($cfg, $this->order, $billing, $delivery, $user);
+        $fx = new FX('EUR', 0.2462);
+
+        $this->request = new Request($cfg, $this->order, $billing, $delivery, $user, $fx);
```
Comment (drealecs) [inline tests/PayU/Alu/RequestTest.php]: Do we still have a test for when fx is not used and parameters should not be sent?

Comment (vladx1900) [reply tests/PayU/Alu/RequestTest.php]: Done!

File: src/PayU/Alu/Request.php (hunk 1 above)
Comment (vladx1900) [reply src/PayU/Alu/Request.php]: Done!

File: tests/PayU/Alu/RequestTest.php [hunk 3]
```diff
@@ -15,12 +15,37 @@ class RequestTest extends \PHPUnit_Framework_TestCase
...
+    /**
+     * @var User
+     */
+    private $user;
+
     public function setUp()
     {
 
-        $cfg = new MerchantConfig('MERCHANT_CODE', 'SECRET_KEY', 'RO');
+        $this->cfg = new MerchantConfig('MERCHANT_CODE', 'SECRET_KEY', 'RO');
```
Comment (drealecs) [inline tests/PayU/Alu/RequestTest.php]: I'm not a fan of keeping unnecessary state. Can't you just pass variables to functions, as before?

**Pull request merged**